### DISCUSSION TITLE
1SWEAY23/24_02230290_NAMGAY_WANGCHUK

### DISCUSSION THREAD

**Discussion by Namgay282004** [kevinlin1/just-the-class#44] [Q&A]
https://www.figma.com/file/M89KmhuuxVgkJBQcb373N1/Untitled?type=design&node-id=111-244&mode=design&t=MPx96A7LgPi3CFtT-0

**Comment by kevinlin1** [comment]
Hi, I think you meant to post this in the [discussion for your course](https://github.com/rubcstswe/sda101/discussions). This discussion is for the website template, not any particular course.
